deploy-review/solutions/microservice-session/src/main/resources/application.properties

```diff
@@ -0,0 +1,11 @@
+quarkus.devservices.enabled = false
```

Comment:
In my GE for Service Binding (SB) I disabled  the dev services because I didn't want them to collide with the use of SB on tests. Since SB is not required for the exam, probably we are not going to use it for tests.. so we might wan to enable again the dev services.

Reply:
Does that mean I should set it to true so as to make sure it is enabled.

Reply:
Your application is not using SB for testing, so we can get rid of that flag. Please be sure that all the tests pass when we remove the config setting.